CMakeLists.txt

```diff
@@ -88,15 +103,17 @@ add_subdirectory(src)
 #
 add_subdirectory(srcpy)
 
-#
-# manage the demos
-#
-add_subdirectory(demos)
+if (BUILD_ROS_DYNAMIC_GRAPH)
```

Comment:
the if  should be inside the `demos/CMakeLists.txt` here

Reply:
I addressed this one.